tests/extra_scope_test.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+from bs4 import BeautifulSoup
+import re
+import textwrap
+import pytest
+
+if TYPE_CHECKING:
+    from sphinx.testing.util import SphinxTestApp
+
+
+@pytest.mark.parametrize(
+    "scope_url,expected_scope_url",
+    [
+        (None, "."),
+        ("", ""),
+        ("/", "/"),
+        ("/subsite", "/subsite"),
+    ],
+    ids=[
+        "base_url",
+        "empty",
+        "root",
+        "subsite",
+    ],
+)
+def test_extra_scope(
+    immaterial_make_app,
+    scope_url: str | None,
+    expected_scope_url: str,
+):
+    if scope_url is not None:
+        scope_url = f'"{scope_url}"'
+
+    app: SphinxTestApp = immaterial_make_app(
+        extra_conf=textwrap.dedent(
+            f"""
+            html_theme_options = {{
+                "scope": {scope_url}
+            }}
+            """
+        ),
+        files={
+            "index.rst": "Sample text",
+        },
+    )
+    app.build()
+
+    with open(app.outdir / "index.html", mode="r") as file:
```

Comment:
older versions of Sphinx (`<7.2`) treated `outdir` like a `str`. The following mypy warning is telling you this as well:
```
nox > mypy 
tests/extra_scope_test.py:50: error: Unsupported left operand type for /
("str")  [operator]
        with open(app.outdir / "index.html", mode="r") as file:
                  ^~~~~~~~~~~~~~~~~~~~~~~~~
Found 1 error in 1 file (checked 94 source files)
```
```suggestion
    with open(pathlib.Path(app.outdir) / "index.html", mode="r") as file:
```

Reply:
Fixed